src/app/signup/signup.component.ts

```diff
@@ -49,4 +51,9 @@ export class SignupComponent implements OnInit {
         },
       );
   }
+
+  // Captcha error
+  onError(error: any) {
```

Comment:
Same as above: rename to onCaptchaError?

Reply:
done